components/infobox/wikis/starcraft/infobox_map_custom.lua

```diff
@@ -114,14 +109,7 @@ end
 ---@param args table
 ---@return string[]
 function CustomMap:getWikiCategories(args)
-	local players = args.players
-	if String.isEmpty(players) then
-		players = CustomMap:_tlpdMap(args.id, 'players')
-	end
-
-	if String.isEmpty(players) then
-		return {}
-	end
+	local players = args.players or self:_tlpdMap(args.id, 'players')
```

Comment:
```suggestion
	local players = args.players or self:_tlpdMap(args.id, 'players')
	if String.isEmpty(players) then return {} end
```
could still be nil, so to avoid concatenation with nil ...